content/authorization.md

```diff
@@ -382,6 +383,15 @@ where the audience is provided:
 &aud=https%3A%2F%2Ffhir-ehr.sandboxcerner.com%2Fdstu2%2F0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca%2F
 </pre>
 
+#### Redirect URI ####
+
+When performaing an authorization grant request Cerner requires the application to 
```

Comment:
It is not required when requesting the authorization code, it is optional, if not provided will use registered one

Reply:
+1 - if they have multiple redirects, they should specify, or it will use the primary one configured.

Reply:
@daliboz  - That is correct, we will redirect to the primary redirect URI, not the additional redirect URIs that are registered.

Reply:
Addressed : https://github.com/cerner/fhir.cerner.com/pull/370/commits/85110eda39d7034a9be61302a9c2251f700a489c